lib/Lua/Raw.pm6

```diff
@@ -177,7 +177,6 @@ has $.lib = do {
     warn "Attempting to use unsupported Lua version '$lib'; this is likely to fail"
         unless $lib eq <5.1 jit-5.1>.any;
     $lib = "lua$lib";
-    $lib = "lib$lib" unless $*VM.config<dll> ~~ /dll/;
```

Comment:
On my system it added the lib -- maybe this is a change to Rakudo since this was last working.